src/text-tags.js

```diff
@@ -22,19 +22,33 @@ function buildTagsFromStory(config, story, url = {}, data = {}) {
 
   const seo = story.seo || {};
 
-  const storyUrl = story.url || `${config["sketches-host"]}/${story.slug}`;
+  let descriptionWithExtraFallback = "";
+  let ogDescriptionWithExtraFallback = "";
+  const enableMetaDescriptionFallback = get(seoConfig, ["enableMetaDescriptionFallback"], false);
```

Comment:
why put this behind a toggle?

Reply:
we don't want to affect the custom publishers